packages/foundry/contracts/prediction-market/lib/PositionStorage.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { 
+    Position,
+    PredictionMarket 
+} from '../Types.sol';
+
+import { Math } from "@openzeppelin/contracts/utils/math/Math.sol";
+
+import { IERC20Metadata } from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+library PositionStorage { 
+
+    /// @notice the user position does not have enough balance to apply the given delta
+    error InsufficentPositionBalance(bytes32 market, address user, uint256 balance, int256 delta);
+
+    /**
+     * @notice get the claimable token balance from the market
+     * @dev technically we can pass markets that are not closed here and be okay since the closing values
+     * for the bull/bear sides would not have been set.
+     * @param self The user position
+     * @param market the closed market
+     */
+    function getClaimableBalance(
+        Position memory self,
+        PredictionMarket memory market
+    ) internal returns (uint256 claimAmount) {
+        uint256 claimTokenMultiplier = 10**IERC20Metadata(market.token0).decimals();
```

Comment:
Could use documentation here. Could also pre-compute this when the market is created, and store it as a data field.